test/functional/page_objects/header_page.js

```diff
@@ -162,10 +162,12 @@ export function HeaderPageProvider({ getService, getPageObjects }) {
       .findByLinkText(quickTime).click();
     }
 
-    async getToastMessage() {
-      remote.setFindTimeout(defaultFindTimeout);
-      return await remote.findDisplayedByCssSelector('kbn-truncated.toast-message.ng-isolate-scope')
-      .getVisibleText();
+    async getToastMessage(findTimeout = defaultFindTimeout) {
+      await remote.setFindTimeout(findTimeout);
```

Comment:
Something to keep in mind for future PRs, it's be nice to add stuff like this to the [`find` service](https://github.com/elastic/kibana/blob/master/test/functional/services/find.js#L16-L23)